custom_model_runner/datarobot_drum/drum/enum.py

```diff
@@ -391,13 +391,17 @@ class TargetType(Enum):
     MULTICLASS = "multiclass"
     TRANSFORM = "transform"
     TEXT_GENERATION = "textgeneration"
+    GEO_POINT = "geopoint"
```

Comment:
Based on in-line comment for this class `TargetType` and once you make more development progress for geo point type, make sure to update root-level documentation:
```
MODEL-METADATA.md
```